lib/helper.js

```diff
@@ -225,6 +226,86 @@ class Helper {
   static isNumber(obj) {
     return typeof obj === 'number' || obj instanceof Number;
   }
+
+  /**
+   * @static
+   * @param {function(?)} nodeFunction
+   * @throws {TypeError} - Throws error if argument isn't a function.
+   * @return {function(?):!Promise<?>}
+   * @example
+   *   const writeFileAsync = Helper.promisify(require('fs').writeFile);
+   *   ...
+   *     await writeFileAsync(...);
+   */
+  static promisify(nodeFunction) {
+    if (typeof nodeFunction !== 'function')
+      throw new TypeError('A function is expected');
+
+
+    // If code run under NodeJS v8+, then by util.promisify.custom key
+    // may exist promisified version of function.
+    // see: https://nodejs.org/dist/latest-v8.x/docs/api/util.html#util_custom_promisified_functions
+    if (util.promisify && util.promisify.custom) {
+      if (nodeFunction[util.promisify.custom])
+        /**
+         * @param {!Array<?>} args
+         * @return {!Promise<?>}
+         */
+        return util.promisify(nodeFunction);
+    }
+
+    // Because native util.promisify differently work with arguments
+    // we can't use this function as ponyfill
+
+    /**
+     * @param {!Array<?>} args
+     * @return {!Promise<?>}
+     */
+    function promisified(...args) {
+      return new Promise((resolve, reject) => {
+        function callback(err, ...result) {
+          if (err)
+            return reject(err);
+
+          // If callback invoked with single argument, we return in in Promise directly
+          if (result.length === 1)
+            return resolve(result[0]);
+          else
+            // Otherwise return array of arguments
+            return resolve(result);
+        }
+        args.push(callback);
+        nodeFunction.call(null, ...args);
+      });
+    }
+
+    promisified.__isPromisified__ = true;
+    return promisified;
+  }
+
+  /**
+   * @description Add to object for each method promisified analog by ${methodName}Async
+   * @static
+   * @param {Object} object
+   * @example
+   *  const fs = Helper.promisifyAll(require('fs'));
+   *  fs.writeFileAsync(...); // -> Promise
+   */
+  static promisifyAll(object) {
```

Comment:
we don't need this method

Reply:
Well, I can remove it, but it may be needed in the future.

Reply:
Thanks, we'll add it if/when the need arises.